### PR TITLE
docs(zh): translate publish to npm

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

translate publish to npm

### CODE REVIEW

File: src/pages/zh-cn/guides/publish-to-npm.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+{
+  "exports": {
+    ".": "./index.js",
+    "./astro": "./MyAstroComponent.astro",
+    "./react": "./MyReactComponent.jsx"
+  }
+}
+```
+
+在这个示例中，导入 `my-component` 会使用 `index.js`。而当导入 `my-component/astro` 或 `my-component/react` 则会分别使用 `MyAstroComponent.astro` 和 `MyReactComponent.jsx`，
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/guides/publish-to-npm.md:147]: Small typo
```suggestion
在这个示例中，导入 `my-component` 会使用 `index.js`。而当导入 `my-component/astro` 或 `my-component/react` 则会分别使用 `MyAstroComponent.astro` 和 `MyReactComponent.jsx`。
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for your work @kecrily! LGTM. I just spotted a small typo and I pointed out something you may want to update now or later.

File: src/pages/zh-cn/integrations/integrations.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Astro 社区的成员已经成功地将多个第三方工具和服务集成到
 
 以下是来自社区的一些生产站点、存储库、博客文章和视频，展示了如何将 Astro 与各种流行的 CMS、电子商务、身份验证/授权、搜索和评论技术连接起来。
 
-> 寻找特定的集成？查看我们新的[集成库](https://astro.build/integrations)，并了解如何[将您的集成添加](/zh-cn/guides/publish-to-npm/#integrations-library)到库中！
+> 寻找特定的集成？查看我们新的[集成库](https://astro.build/integrations)，并了解如何[将您的集成添加](/zh-cn/guides/publish-to-npm/#集成库)到展示板中！
```
[the remark this answers is not in the document]
Comment (kecrily) [reply src/pages/zh-cn/integrations/integrations.md:11]: Update it right now is fine

[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply src/pages/zh-cn/integrations/integrations.md:11]: This tip uses a custom label, so I can't update it myself since I don't know Chinese 😅.

**Pull request merged**